viz/charts/linkerd-viz/templates/web-policy.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: policy.linkerd.io/v1alpha1
+kind: Server
+metadata:
+  namespace: {{.Values.namespace}}
+  name: web
```

Comment:
should this be called web-admin or something?

Reply:
Indeed!